week8_homework/api.js

```diff
@@ -0,0 +1,55 @@
+// Aici tinem request-urile la server
+
+//Preia todos de la server
+
+const baseURL = "http://localhost:3000";
+
+//Functia pentru 1. Aici aducem todo-urile din server
+function getTodos(){
+    return fetch(`${baseURL}/todos`)
+        .then(function(response){
+            return response.json()
+        })
+}
+
+//Functie pentru 2. Cu ajutorul functiei createTodo trimitem catre 
+//server todo-ul creat
+function postTodo(newTodo){
+    return fetch(`${baseURL}/todos`, {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(newTodo)
+    })
+}
+
+//Functia pentru 4. Update todo
+function updateTodo(todo, check) {
+    if(check === false){
+        const updateTodoId = todo.id;
+        const newObjectTodo = todo;
+        newObjectTodo.completed = true;
```

Comment:
in loc de if(check === false){ si sa duplici atata cod... nu puteai face simplu  `newObjectTodo.completed = !check` ?

Reply:
Nu am fost deloc eficient :) , am corectat la check si am pus checkboxes in loc de buttons.
Multumesc de corectari!